org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -800,6 +800,54 @@ GithubOidcOpenChallengesDeploy:
       - !Ref ITSandboxAccount
     Region: us-east-1
 
+# Permissions to push an image to private registry
+# See: https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-push-iam.html
+GithubOidcNbConvertDeploy:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+  StackName: !Sub ${resourcePrefix}-${appName}-nbconvert-deploy
+  Parameters:
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-nbconvert-deploy
+    MaxSessionDuration: 7200
+    PolicyDocument: >-
```

Comment:
why not just give the CI the `AmazonEC2ContainerRegistryFullAccess` managed policy?

Reply:
I originally thought about giving more permissions in case I want to deploy the lambda itself from GH CI then decided to clamp it down since I only want to push the image from GH CI (and deploy the lambda from the regular Synapse CI).
It would simplify management...